components/omega/doc/design/PGrad.md

```diff
@@ -0,0 +1,224 @@
+(omega-design-pressure-grad)=
+# Pressure Gradient
+
+
+## 1 Overview
+The pressure gradient will be responsible for computing the horizontal gradients of both the pressure and geopotential terms for the non-Boussinesq primitive equations implemented in Omega.
+In the non-Boussinesq model, the conserved quantity is mass rather than volume.
+In Omega the prognostic variable $\tilde{h}$ is a pseudo thickness, rather than geometric thickness in m as in a Boussinesq model.
+ Some non-Boussinesq models are written in pressure coordinates (e.g. [de Szoeke and Samelson 2002](https://journals.ametsoc.org/view/journals/phoc/32/7/1520-0485_2002_032_2194_tdbtba_2.0.co_2.xml).
+ However, Omega is written in general vertical coordinates and can reference either pressure $p$ or distance $z$ in the vertical.
+In a pure pressure coordinate the pressure gradient term disappears (since the pressure does not vary along lines of constant pressure), just as how the geopotential term disappears in a pure z coordinate model.
+However, similar to MPAS-Ocean's support for tilted height coordinates, Omega will allow for tilted pressure coordinates.
+This means that Omega will need to compute both the pressure and geopotential gradients.
+
+## 2 Requirements
+
+### 2.1 Requirement: Support for tilted pressure coordinates for the non-Boussinesq primitive equations
+
+The pressure gradient will compute the horizontal gradients of both the pressure and geopotential to support tilted pressure coordinates in the non-Boussinesq model.
+This will allow for the use of a $p^\star$ coordinate, which functions similarly to the $z^\star$ in the Boussinesq MPAS-Ocean model.
```

Comment:
will this be updated to the \tilde{z} coordinate instead of $p^\star$?  I guess psuedo height could be thought of as $p^\star$ like so maybe no need to change anything

Reply:
$\tilde{z}*$? 😝

Reply:
I was hoping we could continue calling it p* since \tilde{z}^* seems confusing.